RedBeanPHP/QueryWriter.php

```diff
@@ -477,7 +477,7 @@ public function esc( $databaseStructure, $dontQuote = FALSE );
 	 *
 	 * @return void
 	 */
-	public function wipeAll();
+	public function nuke();
```

Comment:
I'm afraid this will break people's code because they assume different behavior in QueryWriter. QueryWriter API is supposed to be stable.

Reply:
I know how you feel regarding BC but in order to harmonize things I think it's a very small inconvenience.
I mean, once this PR is applied people just have to change all their "wipeAll" (if they are using it, which shouldn't be a lot of people) to "nuke".
But I guess we could leave it as it currently is and add a new nuke function instead...

Reply:
What about adding wipeAll() with a parameter to drop tables as well. Then nuke() can be an alias for wipeAll( TRUE );